concepts/05 Widgets/Sankey/05 Nodes and Links/05 Colorize Nodes and Links.md

```diff
@@ -39,6 +39,51 @@ When the palette does not have enough colors to paint each node differently, it
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template> 
+        <DxSankey
+            palette-extension-mode="alternate"
+            palette="Bright"
+            /* or a custom palette */
+            /* :palette="['#70c92f', '#f8ca00', '#bd1550']" */
+        >
+        </DxSankey>
```

Comment:
First of all, I suggest that you make this component self-closing - <dxSankey />. In this example, it doesn't contain any nested components. After that, let's make the same changes in this PR and in the [dxFunnel](https://github.com/DevExpress/devextreme-documentation/pull/973/files) snippets.

Then I would like to change the comments - they are made in the markup, so let's use the following style:

<DxSankey
    palette-extension-mode="alternate"
    palette="Bright"
/>  <!-- or a custom palette -->
      <!-- :palette="['#70c92f', '#f8ca00', '#bd1550']" -->